bondpy/bondpy/bondpy.py

```diff
@@ -91,10 +93,11 @@ class Bond(object):
     #           the sister's end
     # \param on_broken callback that will be called when the bond is broken.
     # \param on_formed callback that will be called when the bond is formed.
-    def __init__(self, topic, id, on_broken=None, on_formed=None):
+    def __init__(self, topic, id, node, on_broken=None, on_formed=None):
```

Comment:
I think it would be a bit nicer to have `node` be the first argument; that more closely matches how we do things elsewhere.

Reply:
Changed in [4949755](https://github.com/ros/bond_core/pull/89/commits/4949755d949cb878b38efabff13963561a70a10d)